content/applications/inventory_and_mrp/quality/quality_check_types/picture_check.rst

```diff
@@ -124,33 +125,47 @@ specified in the :guilabel:`Work Order Operation` field on the |QCP| form. If a
+open the *Shop Floor* module.
+
+When accessed from a specific work order, the *Shop Floor* module opens to the page for the work
+center where the order is configured to be processed, and isolates the work order's card so that no
+other cards are shown.
 
-With tablet view open, complete the steps listed on the left side of the screen until the *Take a
-Picture* quality check step is reached. Upon reaching the check, follow the instructions for how to
-take the picture, which appear at the top of the screen.
+Process the work order's steps until the *Take a Picture* quality check step is reached. Click on
+the step to open a pop-up window that includes instructions for how the picture should be taken.
+After taking the picture, make sure that it is stored on the device being used to process the
+quality check (computer, tablet, etc.).
 
-After taking the picture, make sure it is stored on the device being used to process the work order
-(computer, tablet, etc.). Then, click the :guilabel:`Take a Picture` button to open the device's
-file manager. In the file manager, navigate to the picture, select it, and click :guilabel:`Open` to
-attach it. Finally, click :guilabel:`Validate` to complete the quality check.
+Then, click the :guilabel:`Take a Picture` button on the pop-up window to open the device's file
+manager. In the file manager, navigate to the picture, select it, and click :guilabel:`Open` to
+attach it.
 
-.. image:: picture_check/work-order-picture-check.png
+Finally, click :guilabel:`Validate` at the bottom of the pop-up window to complete the quality
+check. The pop-up window will then move on to the next step of the work order.
+
+.. image:: picture_check/picture-check-shop-floor.png
    :align: center
-   :alt: A Take a Picture check for a manufacturing work order.
+   :alt: A Take a Picture check in the Shop Floor module.
+
+If a quality alert must be created, exit the pop-up window by clicking the :guilabel:`X` button in
+the top-right corner. Then, click the :guilabel:`⋮ (three vertical dots)` button on the bottom-right
+corner of the work order card to open the :guilabel:`What do you want to do?` pop-up window.
 
-If a quality alert must be created, do so by clicking the :guilabel:`☰ (three horizontal lines)`
-button in the tablet view, and selecting :guilabel:`Quality Alert` from the :guilabel:`Menu` pop-up
-window. A :guilabel:`Quality Alerts` pop-up window appears, from which a quality alert can be
-created. For a complete guide on how to fill out quality alert forms, view the documentation on
-:ref:`quality alerts <quality/quality_management/quality-alerts>`.
+On the :guilabel:`What do you want to do?` pop-up window, select the :guilabel:`Create a Quality
+Alert` button. Doing so opens a blank quality alert form in a new :guilabel:`Quality Alerts` pop-up
+window. For a complete guide on how to fill out quality alert forms, view the documentation on
+:doc:`quality alerts <../quality_management/quality_alerts>`.
```

Comment:
```suggestion
window.

For a complete guide on how to fill out quality alert forms, view the documentation on :doc:`quality
alerts <../quality_management/quality_alerts>`.
```

Reply:
a little separation between the instructions and this suggestion to check out other documentation add a bit of organization to the overall doc.